reviews/sandbox/2022-wasmedge-annual.md

```diff
@@ -0,0 +1,155 @@
+* [version 0.12.1](https://github.com/WasmEdge/WasmEdge/releases/tag/0.12.1)
+* [version 0.13.0](https://github.com/WasmEdge/WasmEdge/releases/tag/0.13.0)
+
+
+With each release, WasmEdge delivers important features that are important to cloud-native users.
+
+* Advanced networking with the WasmEdge sockets library. It supports non-blocking sockets, DNS, TLS. That allows WadmEdge to run modern HTTPS client and server applications. 
+* Support for the Rust Tokio library. It allows many Tokio-based async Rust applications, especially async networking apps, to run seamlessly on WasmEdge.
+* Container and K8s tooling. WasmEdge is an OCI-compliant runtime, so it could be managed and orchestrated by K8s and K8s tooling, like crun, youki, runwasi, containerd, CRI-O, K8s, Kind, OpenYurt, KubeEdge, SuperEdge.
+* An officil package in Fedora and Red Hat Enterprise Linux (RHEL) distributions. 
+* JavaScript support. WasmEdge now supports the full JavaScript language spec as well as common JavaScript module systems, including ES6, CommonJS (CJS), and NPM modules.
+* Support for WebAssembly feature extensions such as WASI-NN and WASI-Crypto.
+* Support for optional WebAssembly specs and proposals from the community, including the SIMD proposal, the Tail Call proposal, the extended-const proposal, the multi-memories proposal, the Import/Export mutable globals proposal, the Non-trapping float-to-int conversions proposal, the Sign-extension operators proposal, the Multi-value proposal, the Reference Types proposal, the Bulk memory operations proposal and the threads proposal.
+* Improved cross-platform compatibility. WasmEdge added support for macOS, Windows, Andriod, seL4, OpenWRT, RISC-V, and OpenHarmony operating systems, as well as ARM32, ARM64, Apple M1 and M2 hardware.
+* Enhanced host SDKs in C, Rust, and Go languages.
+* A host / Wasm data binding toolkit called the `wasmedge-bindgen`.
+* A new plugin system that makes WasmEdge native host functions and extensions easier to develop and install.
+* A lot of performance improvements.
+
+The WasmEdge community has expanded its collaboration with many open-source projects in the cloud-native space in the last year. Some of these projects now have production customers using WasmEdge.
+
+* OpenFunction (CNCF project) supports WasmEdge as a container runtime.
+* WebAssembly Languages Runtime, maintained by VMWare, supports WasmEdge as a runtime
+* LF Edge eKuiper and shifu uses WasmEdge to process data streamed from IoT devices.
+* Kuasar supports WasmEdge as a container runtime.
+* containerd's runwasi supports WasmEdge as a container runtime.
+* youki, the OCI Container Runtime, supports WasmEdge as a container runtime.
+* Essa-rs ueses WasmEdge to execute serverless functions on the essa-rs platform.
+* The OpenGauss and libsql communities used WasmEdge to power UDFs (User Defined Functions) in database queries.
+* Docker Desktop integrates WasmEdge to support Wasm containers.
```

Comment:
Since the point is regarding collaboration with open-source projects, Docker Desktop is not open-source, perhaps this could be replaced if the intent of the different projects was in reference with the above line for the collaboration with open source projects specifically